tools/maven/checkstyle.xml

```diff
@@ -198,7 +198,7 @@ This file is based on the checkstyle file of Apache Beam.
       <!-- Checks for out of order import statements. -->
       <property name="severity" value="error"/>
       <!-- Flink imports first, then other imports, then javax, java and scala imports, then static imports. -->
```

Comment:
nit: Update this comment line as well?
"Flink and Flink shaded imports first, then other ..."